### ISSUE TITLE
No errors but no code completion either

### ISSUE THREAD

**Issue by shadycuz** [redhat-developer/vscode-java#1978]
I can not find any errors, things seem to work, but I do not get any code completion / intellisense help.

##### Environment
- Operating System: WSL Ubuntu
- JDK version: 1.8 for code and 1.11 for plugin
- Visual Studio Code version: 1.57.0
- Java extension version: v0.79.2

##### Steps To Reproduce
1. Follow all the get starting guides
2. Follow all the debugging guides.
3. Try different things for a couple months
4. Realize the "Discussion" you opened for help has been removed.
5. Create issue asking for help.

[My project](https://github.com/DontShaveTheYak/jenkins-std-lib)
[Client Log](https://gist.github.com/shadycuz/76e2a8fc1141ba82626b83ddf271e7a0)
[Server log](https://gist.github.com/shadycuz/bfc710c81489ea67411fb9a023216932)

##### Current Result
No auto-completion
##### Expected Result
Auto-completion 
##### Additional Informations
![image](https://user-images.githubusercontent.com/17090999/121772236-2f7e0480-cb42-11eb-854d-4e9cce1907ae.png)
![image](https://user-images.githubusercontent.com/17090999/121772254-49b7e280-cb42-11eb-9747-027435ee436c.png)

**Comment by shadycuz** [comment]
I'm using groovy.... so I created a java file and wow everthing seems to be working. Seems like this plugin should see that someone is using groovy and add the association to .groovy. I know the syntax is not perfectly the same, but I was just looking for some autocompletion for variable names and stuff like that. I set this in my settings:

```
"files.associations": {
        "Jenkinsfile": "groovy",
        "**.groovy": "groovy, java"
    },
```

and I still could not get the plugin to turn on unless I was using a .java file extention. 

After playing around it does seem like even if I could get it to work it would not be that helpful for groovy =/
